draft-ietf-mls-protocol.md

```diff
@@ -1960,16 +1983,11 @@ keys for nodes in its direct path that are being reset by the corresponding
 Commit.
 
 ~~~~~
-struct {
-    HPKEPublicKey public_key;
-    opaque unmerged_leaves<0..2^32-1>;
-    optional<ClientInitKey> client_init_key;
-} RatchetNode;
-
 struct {
   // GroupContext inputs
   opaque group_id<0..255>;
   uint64 epoch;
+  opaque root_node_hash<0..255>;
```

Comment:
This is a good starting point, we should see if we need something fancier in the future, like a Merkle tree (or reuse the node hashes).